static/src/inline-svgs/icon/cross-red.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15.5 21"><style>.st0{fill:#CC2B12;}</style><path class="st0" d="M3.6 21c-.1 0-.1-.1-.2-.1-.4-.3-.9-.3-1.4-.4-.9-.2-1.2-1-1.6-1.7-.2-.3-.3-.6-.4-.9v-1.1c.2-.2.3-.5.6-.7.7-.6 1.2-1.2 1.6-2 .1-.2.2-.5.4-.7.2-.3.5-.6.7-.9v-.2c.1-.4 0-.9.5-1.2.2-.2.3-.8.1-1-.7-.5-1-1.1-1.4-1.6-.6-.6-.9-1.4-1.4-2.1-.1-.2-.2-.5-.3-.8 0-.2-.1-.5-.2-.7-.4-.4-.2-.8 0-1 .2-.3.4-.7.9-.8.4-.1.7.2 1.1.3.8.3 1.5.7 2 1.4.5.6 1 1.2 1.6 1.7.1.1.4.3.5 0 .2-.4.5-.7.7-1.1.1-.2.3-.4.4-.6.2-.1.1-.4.2-.5.5-.3.9-.6 1-1.1 0-.1.3-.2.3-.3 0-.5.3-.7.6-1 .1-.1.3-.1.3-.2.5-.5 1.1-1 1.6-1.5.2-.2.5-.3.8-.1l.6.3c.2.1.5.2.7.4.3.2.5.5.8.7.4.4.7.8.7 1.4 0 .2-.2.3-.3.5-.2.3-.1.8-.5 1.1-.2.1-.3.4-.4.6-.1.2-.2.4-.3.5.1.2-.1.4-.2.5-.1.2-.4.2-.5.4-.1.1 0 .4 0 .5-.5 0-.5 0-.7.7.1 0 .2 0 .2.1-.1.3-.2.5-.3.8-.1.1-.1.2-.2.3-.2.2-.4.6-.6.6-.4-.1-.5.2-.6.4-.1.2-.1.5-.2.7-.1.2-.1.6-.2.7-.5.3-.2.6 0 .8.4.4.9.7 1.2 1.3.1.3.5.5.7.8.3.3.6.5.9.8.3.3.4.8.7.9.5.2.6.6.9.9.5.5.2 1.2.3 1.8l-.2.2c-.2.2-.4.4-.7.5-.1.1-.2.1-.3.1-.6-.2-1.3.1-1.9-.4h-.1c-.5 0-.9-.2-1.1-.7l-.6-.9c-.7-.7-1.4-1.3-2.1-2-.1-.1-.4-.2-.5-.2-.2.1-.3.4-.4.6-.1.1-.1.2-.1.3l-1.2 1.5c-.1.6-.4.9-.8 1.3-.1.3-.3.5-.3.7-.1.5-.5.7-.9.9-.2.1-.3.3-.5.5z"/></svg>
```

Comment:
Could you move the fill color out of these SVG files, so thay become more reusable and apply the fill as part of the CSS?